portal-api/src/v1/controllers/deal-status/update-submitted-issued-facilities.js

```diff
@@ -48,11 +48,11 @@ const updateSubmittedIssuedFacilities = async (user, deal, auditDetails) => {
       facility.status = CONSTANTS.FACILITIES.DEAL_STATUS.SUBMITTED_TO_UKEF;
     }
 
-    const { data } = await facilitiesController.update(deal._id, facilityId, facility, user, auditDetails);
-
-    return data;
+    await facilitiesController.update(deal._id, facilityId, facility, user, auditDetails);
   });
 
+  await Promise.all(updateFacilitiesPromises);
+
```

Comment:
Will the deal returned in this function include the updated facilities?

Reply:
I'm not sure the previous implimentation did?

Reply:
ie the following
```
const item = { subItems: [{ id: 1 }, { id: 2 }, { id: 3 }] };

item.subItems.forEach((subItem) => {
  return { id: subItem.id + 10 };
});

console.dir({ item }, { depth: null });
// {
//  item: { subItems: [ { id: 1 }, { id: 2 }, { id: 3 } ] }
// }
```

Reply:
Thanks @AlexBramhill I can see it didn't used to so all good